app/views/cases/offender_sar/_partial_case_flags.html.slim

```diff
@@ -0,0 +1,28 @@
+.grid-row
+  .column-full.data-requests
+    h2.request--heading.data-requests__title
+      = t('cases.offender_sar.partial_case_flags_heading')
+
+    - if !case_details.is_partial_case.nil?
+
+      table#data-request-table.report
+        thead
+          tr
+            th Confirmed SSCL COVID-19 partial case
+            th Date letter sent
+            th SSCL managing case
+            th
+        tbody
+          tr
+            td
+              = case_details.is_partial_case? ? "Yes" : "No"
+            td
+              - if case_details.partial_case_letter_sent_dated.present?
+                time datetime="#{case_details.partial_case_letter_sent_dated}"
+                  = l case_details.partial_case_letter_sent_dated, format: :default
+            td
+              - if case_details.further_actions_required.present?
+                = case_details.further_actions_required.humanize
+
+.partial-flags-buttons
```

Comment:
Can we use .button-holder instead (if we need it for margin, if not then remove element)

Reply:
done